### PR TITLE
RFE: add russian translations for man pages

### PR DESCRIPTION
**Pull request by darktemplarbasealt**

### CODE REVIEW

**Comment by darktemplarbasealt** [comment]
Can someone take a look at this pull request please?

File: doc/ru/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#
+# NetLabel Documentation Makefile
+#
+# Author: Paul Moore <paul@paul-moore.com>
```
Comment (pcmoore) [inline doc/ru/Makefile.am]: I'm pretty sure you want your name here and not mine :)

I understand that this is a pretty simple cut-and-paste addition, so if you want you can always mention that this Makefile was based on doc/Makefile.am.

File: doc/Makefile.am [hunk 2]
```diff
@@ -24,6 +24,8 @@ dist_man8_MANS = \
 	man/man8/netlabel-config.8 \
 	man/man8/netlabelctl.8
 
+SUBDIRS = ru
```
Comment (pcmoore) [inline doc/Makefile.am:27]: I wonder if we should move the existing docs to an "en" directory?

**Comment by pcmoore** [comment]
Hi @darktemplarbasealt, thank you for the translations but unfortunately I can't read/write/speak Russian so I'm not sure how much of the content I'll be able to review, but I did have a couple of comments regarding the Makefiles (see above).

Let me see if I can find anyone to help review the translation itself ...

File: doc/ru/Makefile.am (hunk 1 above)
Comment (darktemplarbasealt) [reply doc/ru/Makefile.am]: Sure, changed this. Please let me know if it is still needed to further change it.

**Comment by pcmoore** [comment]
I didn't catch this at first, but as was pointed out on another project, these translations were provided by another person.  In order to consider these patches the original author will need to agree to the "Developer's Certificate of Origin" and provide a sign-off as described in SUBMITTING_PATCHES.

File: doc/ru/Makefile.am (hunk 1 above)
Comment (pcmoore) [reply doc/ru/Makefile.am]: Well, you'll need to update the PR (I can't see your changes), but see my comments about authorship and sign-offs in the main PR comment thread first.

Comment (darktemplarbasealt) [reply doc/ru/Makefile.am]: If you open `Commits` tab or `Files changed` tab you'll see updated files.

Comment (pcmoore) [reply doc/ru/Makefile.am]: Ah thanks, I looked but for some reason missed the date change when I was looking.

**Comment by darktemplarbasealt** [comment]
I've got sign-offs. Please let me know if anything else is needed.

**Comment by pcmoore** [comment]
Thanks for the update.  Unfortunately I haven't found any third-party Russian speakers to help verify that the translations are correct, so we will probably need to verify the translations using Google Translate.  This is far from ideal, but the best we can do I think.

**Comment by darktemplarbasealt** [comment]
Are there any news on this pull request?

**Comment by pcmoore** [comment]
Unfortunately no, I'm sorry.

File: doc/Makefile.am (hunk 2 above)
Comment (pcmoore) [reply doc/Makefile.am:27]: Nevermind, looking at how manpage translations are handled, this looks correct.

File: doc/ru/man/man8/netlabel-config.8 [hunk 3]
```diff
@@ -0,0 +1,49 @@
+.TH "netlabel-config" 8 "31 мая 2013" "paul@paul-moore.com" "Документация по NetLabel"
+.\" //////////////////////////////////////////////////////////////////////////
+.SH ИМЯ
+.\" //////////////////////////////////////////////////////////////////////////
+netlabel-config \- утилита настройки NetLabel
```
Comment (pcmoore) [inline doc/ru/man/man8/netlabel-config.8:5]: Based on what I see from Google Translate, this file looks okay.

File: doc/ru/man/man8/netlabelctl.8 [hunk 4]
```diff
@@ -0,0 +1,200 @@
+.TH "netlabelctl" 8 "31 мая 2013" "paul@paul-moore.com" "Документация по NetLabel"
+.\" //////////////////////////////////////////////////////////////////////////
+.SH ИМЯ
+.\" //////////////////////////////////////////////////////////////////////////
+netlabelctl \- утилита управления NetLabel
```
Comment (pcmoore) [inline doc/ru/man/man8/netlabelctl.8:5]: This file also looks okay based on Google Translate.

**Comment by pcmoore** [comment]
Merged via 11e9476ec7e1cc7a0645db3e19c961ee68829b04, thanks for the translations and your patience!

**Pull request closed without merging**